css/components/cards.css

```diff
@@ -0,0 +1,82 @@
+.card{
+    display: flex;
+    padding: var(--space-sm);
+    border-radius: var(--radius-md);
+    color: var(--background-night-pastel);
+    gap: var(--space-sm);
+    position: relative;
+}
+
+.vertical-card{
+    flex-direction: column;
+    max-width: 18rem;
+}
+
+.horizontal-card{
+    display: flex;
+    max-width: 40rem;
+}
+
+.horizontal-card>.card-image{
+    max-width: 200px;
+}
+
+.card-image{
+    background-color: var(--background-night-pastel);
+    border-radius: var(--radius-md);
+}
+
+.favourite,
+.card-dismiss{
+    color: var(--secondary-font);
+    position: absolute;
+    top: var(--space-md);
+    right: var(--space-md);
+    cursor: pointer;
+}
+
+.fav-selected{
+    color: var(--cherry);
+}
+
+.special-badge{
+    background-color: var(--cherry);
+    color: white;
+    font-size: var(--fs-label);
+    display: inline-block;
+    border-radius: 0 var(--space-xs) var(--space-xs) 0;
+    padding: 0 var(--space-xs);
+    position: absolute;
+    top: var(--space-md);
+}
```

Comment:
this can be extracted to badge component

Reply:
> this can be extracted to badge component

Oh yes, will do it. Thanks for the heads-up